### ISSUE TITLE
Reenable HTF in lts

### ISSUE THREAD

**Issue by andreasabel** [commercialhaskell/lts-haskell#315]
Depends on #313 (haskell-src).

HTF needs to be added __without `--tests`__ since the test-suite does not run properly: https://github.com/skogsbaer/HTF/issues/106.

I tested with
```
stack build --haddock --bench --no-run-benchmarks --stack-yaml stack-lts-8.10.yaml
```

Because of #313, I put the `extra-dep` into `stack-lts-8.10.yaml`:
```
resolver: lts-17.4
flags: {}
packages:
- '.'

extra-deps:
- haskell-src-1.0.3.1
```

**Comment by jkachmar** [comment]
This should be resolved in LTS-17.5; it's currently building and I'll try to remember to update this issue after that's finishes and it's been published.

**Comment by jkachmar** [comment]
[Present in LTS-17.5](https://www.stackage.org/lts-17.5/package/HTF-0.14.0.6)
